exercises/armstrong-numbers/test.scm

```diff
@@ -0,0 +1,91 @@
+(load "armstrong-numbers.scm")
+
+(use-modules (srfi srfi-64))
+
+(test-begin "armstrong-numbers")
+
+(test-eq "Zero is an Armstrong number"
+  (armstrong-number? 0)
+  #t)
```

Comment:
Since these are returning booleans, maybe `test-assert`? https://srfi.schemers.org/srfi-64/srfi-64.html

Reply:
Hmm..., I thought it'd be clearer to the students that we're expecting `#t` or `#f`.  Using `(test-assert ...)` may obfuscate that.  Anyway, I'll change it.  Thanks.

Reply:
Your choice, I think your reasoning makes sense as well